lunatrace/bsl/frontend/craco.config.js

```diff
@@ -19,10 +19,11 @@ module.exports = {
     alias: {},
     plugins: {
       add: [
-        new webpack.ProvidePlugin({
-          process:'process/browser'
-        })
-      ]
+        // new webpack.ProvidePlugin({
```

Comment:
Are these changes intentional? If so, can you leave a comment in here as to why they are commented out?